src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {

```

Comment:
method should not be a static
```suggestion
    public Ball getRandomBall() {
```